sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusMessageBatch.java

```diff
@@ -97,7 +97,7 @@ public int getSizeInBytes() {
      * @throws AmqpException if {@code message} is larger than the maximum size of the {@link
      *     ServiceBusMessageBatch}.
      */
-    public boolean tryAdd(final ServiceBusMessage serviceBusMessage) {
+    public boolean tryAddMessage(final ServiceBusMessage serviceBusMessage) {
         if (serviceBusMessage == null) {
             throw logger.logExceptionAsWarning(new IllegalArgumentException("message cannot be null"));
```

Comment:
This should throw NPE and the error message should be `"'serviceBusMessage' cannot be null"`. Update the JavaDoc too.

Reply:
updated